solr/core/src/java/org/apache/solr/api/V2HttpCall.java

```diff
@@ -397,7 +399,7 @@ protected void handleAdmin(SolrQueryResponse solrResp) {
   @Override
   protected void executeCoreRequest(SolrQueryResponse rsp) {
     if (api == null) {
-      invokeJerseyRequest(cores, core, core.getApplicationHandler());
+      invokeJerseyRequest(cores, core, core.getApplicationHandler(), rsp);
```

Comment:
`rsp` or `solrResp`, but maybe not both ???

Reply:
I found myself confusing the names every time I came into this class.  I held off on renaming variables too much in fear of muddying the diff, but I'm happy to do it here if you think it'd help more than it hurts the review here?

Reply:
;-)   I see, before there was a rsp, so you kept the rsp..     so, you are probably right.  I think it's just my desire for consistency making me notice these nits.   And someone once said "Consistency is the hob goblin of little minds"...  ;-).    I think it's fine.